2022-07-27 14:15:24,474 - No module named 'class_erros'
Traceback (most recent call last):
  File "C:\Users\allan.mesquita\OneDrive - NTT\Documents\Projetos\Automacao_Evidencias\Script\main_qualidade2.py", line 149, in <module>
    resultado = validacao.rec_validation(aba, qtd_linhas)
  File "C:\Users\allan.mesquita\OneDrive - NTT\Documents\Projetos\Automacao_Evidencias\Script\Modulos\validacao.py", line 29, in rec_validation
    from class_erros import Error, SaveError
ModuleNotFoundError: No module named 'class_erros'
